lms/static/sass/views/_homepage.scss

```diff
@@ -28,7 +28,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
       @include transition(all $tmg-f3 linear 0s);
 
       position: relative;
-      border-bottom: 3px solid $action-primary-bg;
+      // border-bottom: 3px solid $action-primary-bg;
```

Comment:
Remove this if you don't need it

Reply:
![course](https://github.com/EduTrigger/edx-platform/assets/46914780/84c5a2b2-abeb-4fae-a03e-5eaa8cc97534)